README.md

```diff
@@ -0,0 +1,147 @@
+
+The Ansible VyOS collection includes a variety of Ansible content to help automate the management of VyOS network appliances.
+
+This collection has been tested against VyOS 1.1.8 (helium).
+
+
+### Supported connections
+The VyOS collection supports ``network_cli`` connections.
+
+## Included content
+
+Click the ``Content`` button to see the list of content included in this collection.
+
+## Installing this collection
+
+You can install the VyOS collection with the Ansible Galaxy CLI:
+
+    ansible-galaxy collection install vyos.vyos
+
+You can also include it in a `requirements.yml` file and install it with `ansible-galaxy collection install -r requirements.yml`, using the format:
+
+```yaml
+---
+collections:
+  - name: vyos.vyos
+    version: 0.0.2
+```
+## Using this collection
+
+
+This collection includes [network resource modules](https://docs.ansible.com/ansible/latest/network/user_guide/network_resource_modules.html).
+
+### Using modules from the VyOS collection in your playbooks
+
+You can either call modules by their Fully Qualified Collection Namespace (FQCN), like `vyos.vyos.vyos_l3_interfaces`, or you can call modules by their short name if you list the `vyos.vyos` collection in the playbook's `collections`, as follows:
+
+```yaml
+---
+- hosts: vyos01
+  gather_facts: false
+  connection: network_cli
+
+  collections:
+    - vyos.vyos
+
+  tasks:
+    - name: Merge the provided configuration with the existing running configuration
+      register: result
+      vyos.vyos.vyos_l3_interfaces: &id001
+        config:
+
+          - name: eth1
+            ipv4:
+
+              - address: 192.0.2.10/24
+            ipv6:
+
+              - address: 2001:db8::10/32
+
+          - name: eth2
+            ipv4:
+
+              - address: 198.51.100.10/24
+            vifs:
+
+              - vlan_id: 101
+                ipv4:
+
+                  - address: 198.51.100.130/25
+                ipv6:
+
+                  - address: 2001:db8::20/32
+        state: merged
+```
+
+
+ The following example task replaces configuration changes in the existing configuration on a VyOS network device, using the FQCN:
+
+ ```yaml
+ ---
+   - name: Replace device configurations of listed static routes with provided
+       configurations
+     register: result
+     vyos.vyos.vyos_static_routes: &id001
+       config:
+
+         - address_families:
+
+             - afi: ipv4
+               routes:
+
+                 - dest: 192.0.2.32/28
+                   blackhole_config:
+                     distance: 2
+                   next_hops:
+
+                     - forward_router_address: 192.0.2.7
+
+                     - forward_router_address: 192.0.2.8
+
+                     - forward_router_address: 192.0.2.9
+       state: replaced
+```
+
+
+
+### See Also:
+
+* [VyOS Platform Options](https://docs.ansible.com/ansible/latest/network/user_guide/platform_vyos.html)
```

Comment:
The platform guide needs to be updated to use FQCN name as well for example change `ansible_connection: network_cli` to `ansible_connection: ansible.netcommon.network_cli`

Reply:
the internal spreadsheet is tracking those changes to platform docs now.